src/main/eo/org/eolang/collections/list.eo

```diff
@@ -265,4 +260,16 @@
           rec-filtered
             (* (index.plus 1) carr)
             func
-            filt-list
\ No newline at end of file
+            filt-list
+
+  # Filter list without index with the function "f".
+  # Here "f" must be an abstract
+  # object with one attribute. The first
```

Comment:
@ivan-egorov42 Can we rephrase this sentence (`The first one for the element`) since there is only one attribute

Reply:
@Graur yes, of course